### ISSUE TITLE
UI issue with the Browse Video page under Settings

### ISSUE THREAD

**Issue by swswsws583** [AerialScreensaver/Aerial#80]
Hi, the Browse Video page isn't showing properly, it's been an ongoing issue from version 1.5.0 - 1.5.2 and in macOS 14 and 15 beta. I need to manually widen the Settings window to a specific width so nothing is overlapping.

<img width="1034" alt="Screenshot 2024-08-06 at 21 37 14" src="https://github.com/user-attachments/assets/95cbe5c5-b121-4818-9e46-44701291591f">
<img width="1381" alt="Screenshot 2024-08-07 at 09 59 53" src="https://github.com/user-attachments/assets/ebf8700b-34cd-47b0-95cd-a73797b76985">

**Comment by glouel** [comment]
Yep can confirm, someone posted some UI fixes but they broke that part, only saw recently. 

I'll put a note to fix it when I have a sec. Thanks for reporting !

**Comment by BenjaminBini** [comment]
I have the same issue. Seems like it only works well when the window have a specific size (the default size? I dont remember if I resized the window or not).

**Comment by glouel** [comment]
This is finally fixed
